Syntax error in scripts/images.sh

A user reported an issue with Ubuntu 18.04.6 (LTS) with one of the scripts. It appears that there is a compatibility issue. 

```
user@node-01:/opt/eseries-perf-analyzer# make run

sh scripts/images.sh

scripts/images.sh: 3: scripts/images.sh: Syntax error: "(" unexpected

Makefile:115: recipe for target 'warn' failed

make: *** [warn] Error 2
```

--- a/Makefile
+++ b/Makefile
@@ -112,5 +112,6 @@ clean: stop rm ## Remove all images and containers built by the project
 
 warn: ##
 ifndef QUIET
-	sh scripts/images.sh
+	chmod +x scripts/*
+	scripts/images.sh
 endif

